docs/getting_started/representation/llm.md

```diff
@@ -479,15 +479,23 @@ To use langchain, you will need to install the langchain package first. Addition
 like openai:
 
 ```bash
-pip install langchain, openai
+pip install langchain
+pip langchain_openai
 ```
 
 Then, you can create your chain as follows:
```

Comment:
Let's also give the simple example without the chain since most users will likely want to make use of the `llm` param.

Reply:
Good catch! I forgot to update the documentation, will fix :)

Reply:
I've updated the documentation with explanations on what to install, the basic usage, and the advanced usage. I also adapted the documentation in the code.